src/App.js

```diff
@@ -1,10 +1,79 @@
-import logo from './logo.svg';
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import { ToastContainer } from 'react-toastify';
+import { useSelector } from 'react-redux';
+import ReservationsList from './components/reservation/ReservationList';
+import Protected from './components/ProtectedRoute';
+import Home from './pages/Index';
+import Login from './pages/Login';
+import Register from './pages/Register';
+import AddYacht from './components/AddYacht';
+import Nav from './components/Nav';
+import DeleteYacht from './components/DeleteYacht';
+import 'react-toastify/dist/ReactToastify.css';
+import Reserve from './components/reservation/Index';
 
 function App() {
```

Comment:
- [ ] Since I wasn't able to run the backend, I can't test the frontend functionality.  This will be taken a look in the next review.